themes/cardinal_service/lib/components/opportunity-row/opportunity_row.scss

```diff
@@ -1 +1,71 @@
 @charset 'UTF-8';
+
+@import '../../scss/config';
+
+.pattern-preview__markup {
+  @include card;
+  position: relative;
+
+  h3 {
+    font-size: 2rem;
+  }
+
+  .su-opportunity-body-content {
+    display: grid;
+
+    @include grid-media-min('md') {
+      grid-template-columns: repeat(3, 33%);
+    }
+  }
+
+  .su-opportunity-row-location,
+  .su-opportunity-row-duration,
+  .su-opportunity-row-open-to {
+    div {
+      margin-left: 30px;
+    }
+
+    &::before {
+      font-family: 'Font Awesome\ 5 Free';
+      font-weight: 900;
+      position: absolute;
+    }
+  }
+
+  .su-opportunity-row-location {
+    &::before {
+      content: '\f3c5';
+    }
+  }
+
+  .su-opportunity-row-duration {
+    &::before {
+      content: '\f073';
+    }
+  }
+
+  .su-opportunity-row-open-to {
+    &::before {
+      content: '\f500';
+    }
+  }
+
+  .su-opportunity-img {
+    position: relative;
+
+    &::after {
+      display: block;
+      content: '';
+      padding-bottom: calc(2 / 3 * 100%);
+    }
+
+    img {
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
```

Comment:
I'm not sure if the way I wrote the ratio to display the event image is the best method/way. Please let me know your thoughts! 🤔